readthedocs/builds/managers.py

```diff
@@ -67,6 +72,12 @@ def create_latest(self, **kwargs):
         defaults.update(kwargs)
         return self.create(**defaults)
 
+    def get_object_or_log(self, **kwargs):
```

Comment:
Please, add a docstring for this method so we know how it's called, what it returns and where or when we call it.